inc/js_scripts/admin/main.js

```diff
@@ -14,27 +11,59 @@
 			} );
 		},
 		render: function () {
-			const _html = $( "#custom-template" ).html();
+			const _html = $( '#post-content-template' ).html();
 			this.$el.html( _.template( _html )( {
 				'option': this.model.get( 'plugin_state' ),
 				'special_word': this.model.get( 'special_word' ),
-				'date_format': this.model.get( 'custom_date_format' )
 			} ) );
+			console.log( this.$el.html() );
 		},
 		events: {
-			'change .state, #special-word, #custom_date_format': 'handleChange',
+			'change .state, #special-word': 'handleChange',
 			'change .color-picker': 'handleColorChange',
-			'click #save-settings': 'saveSettings',
 		},
 		handleChange: function ( event ) {
 			this.model.set( $( event.target ).attr( 'data-model_attr_name' ), $( event.target ).val() );
 		},
 		handleColorChange: function ( event ) {
 			this.model.set( $( event.target ).attr( 'data-model_attr_name' ), $( event.target ).spectrum( "get" ).toHex() );
```

Comment:
```suggestion
			this.model.set( $( event.target ).attr( 'data-model_attr_name' ), $( event.target ).spectrum( 'get' ).toHex() );
```
try to use single-quotes instead of double-quotes where you can